addons/account/models/partner.py

```diff
@@ -42,11 +42,31 @@ class AccountFiscalPosition(models.Model):
     # To be used in hiding the 'Federal States' field('attrs' in view side) when selected 'Country' has 0 states.
     states_count = fields.Integer(compute='_compute_states_count')
     foreign_vat = fields.Char(string="Foreign Tax ID", help="The tax ID of your company in the region mapped by this fiscal position.")
+    foreign_vat_header_mode = fields.Selection(
+        selection=[('templates_found', "Templates Found"), ('no_template', "No Template")],
+        compute='_compute_foreign_vat_header_mode',
+        help="Technical field used to display a banner on top of foreign vat fiscal positions, "
+             "in order to ease the instantiation of foreign taxes when possible."
+    )
 
     def _compute_states_count(self):
         for position in self:
             position.states_count = len(position.country_id.state_ids)
 
+    @api.depends('foreign_vat', 'country_id')
+    def _compute_foreign_vat_header_mode(self):
+        for record in self:
+            if not record.foreign_vat or not record.country_id:
+                record.foreign_vat_header_mode = None
+                continue
+
+            if self.env['account.tax'].search([('country_id', '=', record.country_id.id)], limit=1):
+                record.foreign_vat_header_mode = None
+            elif self.env['account.tax.template'].search([('chart_template_id.country_id', '=', record.country_id.id)], limit=1):
+                record.foreign_vat_header_mode = 'templates_found'
```

Comment:
how do you avoid people clicking again and again on the banner to create taxes for the same country?

Reply:
It's hidden if there are active account.tax objects defined in that country ;) So, if they created them by hand or already instantiated the taxes, no banner.